packages/icons-workflow/bin/build.js

```diff
@@ -117,20 +117,14 @@ async function buildIcons(icons, tag, iconsNameList) {
             id += 'Advert';
         }
 
-        if (id === 'UnLink') {
-            id = 'Unlink';
-        }
-        if (id === 'TextStrikeThrough') {
-            id = 'TextStrikethrough';
-        }
-
-        if (id === 'ChevronDownSize300') {
-            id = 'ChevronDown';
-        }
-
-        if (id === 'CheckmarkSize300') {
-            id = 'Checkmark';
-        }
+        const replacements = {
```

Comment:
nit: could we have only 1 replacements map, maybe outside of the functions? For example, on this one you are missing `github`.